k2/csrc/connect.cc

```diff
@@ -5,15 +5,14 @@
 
 // See ../../LICENSE for clarification regarding multiple authors
 
-#include "k2/csrc/connect.h"
-
 #include <algorithm>
 #include <limits>
 #include <stack>
 #include <unordered_map>
 #include <vector>
 
-#include "glog/logging.h"
+#include "k2/csrc/connect.h"
```

Comment:
The corresponding header file (i.e. `connect.h` here) should be put at the top of `.cc` file.

Reply:
fixed